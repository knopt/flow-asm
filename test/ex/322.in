4
8
9.291091493687233
8.028426243121476 -8.577944688119654 1.1961739733802617 -2.7559132195239737 
-6.604420228778032 4.354012149833274 -5.432702296036278 -6.332625710341202 
9.711294638060174 -8.58184312136024 8.530208889850087 -8.713190439608692 
1.0869416451647673 -4.283693798733694 4.137314304694426 1.9299953978343627 
-7.593871375253034 9.198267350258789 -3.0041756515121643 0.9016761790284136 
-9.411443029011995 7.822584768707863 8.702795268842685 -2.847803567638967 
-6.677792328041834 -9.418643520407812 -1.4820313865881047 5.047700104832879 
9.98510193238516 -4.6919943144898335 -3.400822750875891 3.5687356157828436 

-2.9117861637616667 6.040467489631084 9.79661870702833 -2.708954433556001 -5.579733888475911 7.56218038428911 8.55260865041766 4.167153081264448 
