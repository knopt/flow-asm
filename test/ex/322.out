-248.04351023745443 284.2015575027573 -121.86578928056903 -24.138564737196376 
585.1184577814435 -253.88767547274009 218.33504793964758 221.1848861868222 
-370.72006536643187 429.7100679070939 -478.8614711281145 422.3761993041161 
-15.723937283743496 230.21656272489287 -170.79357110138898 -70.52327126222006 
261.08914160148015 -533.1303482701989 374.1969147145893 41.87301910995941 
393.08427634208846 -577.6686159117429 -366.6384846066934 223.90112037403458 
497.3134023769203 400.5012901533918 58.204650066258004 -187.25584925647712 
-212.19589441908528 69.30828851423719 -53.481673121897856 -94.37201278808342 
